sherpa/image/DS9.py

```diff
@@ -237,8 +237,30 @@ def __init__(self, *args, **kargs):
 
         if doRaise:
             raise RuntimeErr('badwin', _ex)
+
+    elif sys.version_info < (3, 2):
```

Comment:
I think it would only really be a concern if we were trying to still support old versions of Python (pre 2.7), so I'm happy sticking with `sys.version_info` for now :-)